src/components/FormSelectionContainer/FormSelect/FormSelect.tsx

```diff
@@ -13,55 +15,112 @@ interface FormSelectProps {
 
 export const FormSelect: FunctionComponent<FormSelectProps> = ({ id, form, onAddForm, ...props }) => {
   const { config } = useConfigContext();
-  const [errorMsg, setErrorMsg] = useState<string | null>(null);
-  const [isValidDns, setIsValidDns] = useState<boolean>();
+  const [tooltipMsg, setTooltipMsg] = useState<string | null>(null);
+  const [isValidEntry, setIsValidEntry] = useState<boolean>();
+  const [loadingState, setLoadingState] = useState<boolean>(false);
```

Comment:
i think `isValidEntry` and `loadingState` can be combined:

`const [formStatus, setFormStatus] = useState<"initial" | "pending" | "success" | "error">("initial");`